docs/source/conf.py

```diff
@@ -20,13 +23,22 @@
 
 # -- Auto notebook index creation --------------------------------------------
 
-static_indir = "examples/_static"
-rst_outdir = "examples/notebook/"
+example_root_url = "https://delvtech.github.io/elf-simulations-examples/"
+r = requests.get(example_root_url, timeout=5)
```

Comment:
nit: let's try not to ignore pylint whenever possible: rename r to request. and remove the `# pylint: disable=invalid-name` suppression above.

Reply:
This pylint disable exists because of the rest of the script: it's complaining global variables are not all caps, and sphinx is looking for lowercase global variables such as "version" (conf.py originally was generated from sphinx). Regardless, I'll go through and fix single letter variable names for the custom code I added.